### DISCUSSION TITLE
对于使用 Websocket 传输内容的网站，有没有什么已有的方案可供学习？

### DISCUSSION THREAD

**Discussion by marcosteam** [DIYgod/RSSHub#8558] [Q&A]
今天想尝试抓一下有知有行，发现他们的消息推送方式比较清奇，是用 Websocket 传输的：

![image](https://user-images.githubusercontent.com/25745812/142441816-deeafb58-9a42-46e8-abe7-968e496a5ed1.png)

~其实我看到他这种传输方式有点感到不妙了，但是还是想试试~

**Comment by linbuxiao** [comment]
有没有链接呢？

**Reply by marcosteam** [reply]
[可能需登录](https://youzhiyouxing.cn/materials)

**Comment by NeverBehave** [comment]
应该有failback到long poll，尝试触发一下看看效果
